legend-depot-store-metrics/src/main/java/org/finos/legend/depot/store/metrics/MetricsModule.java

```diff
@@ -33,18 +33,18 @@ protected void configure()
         expose(QueryMetricsHandler.class);
     }
 
-
-    @Provides
-    @Singleton
-    @Named("persist-metrics")
-    boolean scheduleMetricsPersistence(SchedulesFactory schedulesFactory, QueryMetricsHandler queryMetrics)
-    {
-        schedulesFactory.register("persist-metrics", LocalDateTime.now().plusMinutes(1), 5 * 60000L, true, () ->
-        {
-            queryMetrics.persistMetrics();
-            return true;
-        });
-        return true;
-    }
+//   TODO: fix persisting metrics to avoid concurrency
```

Comment:
can you remove this from this PR and raise a separate one.. also please dont comment the whole thing just add very long interval for now,  code would look cleaner